### ISSUE TITLE
Lowering not implemented for node: u8[64,3,28,28]{3,2,1,0} aten::as_strided

### ISSUE THREAD

**Issue by tyoc213** [pytorch/xla#2644]
## 🐛 Bug

```
RuntimeError: Error while lowering: u8[64,3,28,28]{3,2,1,0} aten::as_strided
Error: torch_xla/csrc/ir.cpp:247 : Lowering not implemented for node: u8[64,3,28,28]{3,2,1,0} aten::as_strided
```

Wich would be some like this line https://github.com/pytorch/xla/blob/42f4635cd3c8f07032e0ca6f878e4f3aa436ed22/torch_xla/csrc/ir.cpp#L247

## To Reproduce

Well, I have a pretty bad code (not cleaned https://colab.research.google.com/drive/1F4W1tKCKESN2bbBG5IGm9yR_wWJthMBV)  because I was not going to report cause I'm in the middle of changing/testing things and seeing how they affect code... and found this, but I think it is probably more of an error on my part/tests, just reporting to see if it is really a bug.

Was thinking of reproduce it like this  https://github.com/pytorch/xla/issues/2238 but was unable to make a minimal example:

```
import torch
import torch_xla.core.xla_model as xm

z=torch.rand(64,3,28,28)
# just extra try... but doesnt break
device = xm.xla_device()
z = z.to(device)
strided_z = z.as_strided(size=[64,3,28,28], stride=[3,2,1,0])
strided_z
```

## Environment

 - Reproducible on XLA backend [CPU/TPU]: TPU
 - torch_xla version: 20201001

**Comment by tyoc213** [comment]
Also if I do `z=torch.randint(0, (64,3,28,28))` it breaks completely the session without log. (just trying to reproduce minimal error of the above thing)

**Comment by tyoc213** [comment]
[debug_run_strided.tar.gz](https://github.com/pytorch/xla/files/5580449/debug_run_strided.tar.gz)

Counter: aten::as_strided
  Value: 1

**Comment by JackCaoG** [comment]
@tyoc213 I am not sure about the `u8` problem, if you found a way to repo it please let me know.

Regarding the second aten counter you saw, it came from this check
https://github.com/pytorch/xla/blob/403919d167efc64e6d86b269fdd1e2c4a69903de/torch_xla/csrc/aten_xla_type.cpp#L600
which calls 
https://github.com/pytorch/xla/blob/85f48c6f80574e73cffe554c9550776fb9d1079c/torch_xla/csrc/ops/as_strided.cpp#L75

Basically pt_xla does not support 0 stride so it fallback to cpu implementation.

**Comment by tyoc213** [comment]
For the `u8` you just need to run this https://colab.research.google.com/drive/1F4W1tKCKESN2bbBG5IGm9yR_wWJthMBV#scrollTo=Obb_HBYU4wM0 and I until the "Error here" 

![image](https://user-images.githubusercontent.com/506234/100014703-60f8d380-2d9c-11eb-8307-260a375200c4.png)

If I reach the minimal sample, I will provide, but for the moment you need to run all the nb.

**Comment by JackCaoG** [comment]
Hmm, I am consistennly getting 
```
Exception in device=TPU:5: torch_xla/csrc/tensor.cpp:1534 : Check failed: program_shape.parameters_size() == po_data->parameters_data.size() (36 vs. 35)
```
by running your colab, which reminds me of https://github.com/pytorch/xla/issues/2392.

**Comment by tyoc213** [comment]
Will read that one,but I think the error comes from  getstate and setstate and with correct impl the assert pass (I already have a version that pass OK). In this one I just [put get](https://colab.research.google.com/drive/17o09xqEi9eHyQTAi-2iNdlMNnZemhnuY?usp=sharing) but also got 

```
Exception in device=TPU:0: Error while lowering: u8[64,3,28,28]{3,2,1,0} aten::as_strided
Error: torch_xla/csrc/ir.cpp:247 : Lowering not implemented for node: u8[64,3,28,28]{3,2,1,0} aten::as_strided
```

By the way, I use the free colab session, dont know if that matters, but I have seen some times that I dont get it, that is why I was wondering if it is "random" behaviour? maybe a specific sync thing to happen.

**Comment by tyoc213** [comment]
@JackCaoG it is OK if stale?

**Comment by JackCaoG** [comment]
@tyoc213 I am still not able to repo the error you reported. The `ERROR HERE` prints 
```
launching n procs
````

and then just stuck. Were you still able to see the error consistently running your colab?
